test/elixir-mode-font-test.el

```diff
@@ -125,7 +141,8 @@ end"
   (elixir-test-with-temp-buffer
       ":oriole
 :andale
-:ms2pid"
+:ms2pid
+:CapitalizedAtom"
```

Comment:
Bad merge conflict resolution on my part. 
I had removed the line `(should (eq (elixir-test-face-at 26) 'elixir-atom-face))` and now I see that was not a good move. 

Just added it again.